src/components/LocaleSwitcher.astro

```diff
@@ -0,0 +1,27 @@
+---
+import Select from 'fulldev-ui/components/Select.astro'
+import fulldevConfig from 'virtual:fulldev-ui/config'
+---
+
+{
+  Astro.props.locales && (
+    <Select
```

Comment:
Aangezien het een select is, geen switch, lijkt LocaleSelect mij een betere naam